plugins/builtin/target/gce-mig/main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/nomad-autoscaler/plugins"
+	"github.com/hashicorp/nomad-autoscaler/plugins/builtin/target/gce-mig/plugin"
+)
+
+func main() {
+	plugins.Serve(factory)
+}
+
+// factory returns a new instance of the AWS ASG plugin.
```

Comment:
copy/paste remnants?
```suggestion
// factory returns a new instance of the Google Cloud Engine MIG plugin.
```